parser/misc.go

```diff
@@ -584,7 +583,7 @@ func (s *Scanner) isTokenIdentifier(lit string, offset int) int {
 		}
 	}
 
-	checkBtFuncToken, tokenStr := false, hack.String(data)
```

Comment:
Will this influence performance?

Reply:
Perhaps. 
When parser is moved into a separate repository, optimize its performance would be another task. I suggest that we can get the first step done.